src/metabase/models/permissions.clj

```diff
@@ -418,20 +428,7 @@
        (when config/is-test?
          (throw e))))))
 
-; Audit Permissions helper fns
-
-(defn check-audit-db-permissions
```

Comment:
Also not part of this fix, but LSP informed me that this function was not actually used anywhere, so I went ahead and removed it.